food-truck-api/src/main/resources/application.properties

```diff
@@ -1,4 +1,4 @@
-spring.profiles.active=prod
+spring.profiles.active=dev
```

Comment:
Was this supposed to be changed? It looks like it might break CI/CD, but I'm not sure. @nnazo Can you confirm?